server/routes/player.routes.js

```diff
@@ -0,0 +1,35 @@
+const playerRouter = require('express').Router();
+const Player = require('../models/Player.model.js');
+
+function addAPlayer(request, response, next) {
+  console.log('Incoming', request.body);
+
+  if(!request.body) {
+    let err = new Error('You must provide a player');
+    err.status = 400;
+    next(err);
+    return;
+  }
+
```

Comment:
we'll need at some point validation here to fail if 
a) name or email address property is missing or blank
b) whether a player with the same email address exists

...so at the very least we need a comment to remind ourselves or at least, a User Story to handle data validation